app/create_app.py

```diff
@@ -1,44 +1,82 @@
+import os
```

Comment:
file in general is a bit messy, think about how you might break out some of this or group it differently to make it tidier

Reply:
Removed unnecessary on the fly config management